app/models/identity_tijuana/donation.rb

```diff
@@ -6,7 +6,8 @@ class Donation < ApplicationRecord
     has_many :transactions
 
     scope :updated_donations, -> (last_updated_at, exclude_from) {
-      where('donations.updated_at > ?', last_updated_at)
+      includes(:transactions)
+        .where('donations.updated_at > ?', last_updated_at)
```

Comment:
This on the other hand is totally worth keeping as a separate commit, since it documents why that specific change was made. 😎